jdaviz/app.py

```diff
@@ -1022,6 +1022,14 @@ def resize(stack_items):
 
         resize(self.state.stack_items)
 
+        # resize tray items
+        for tray_item in self.state.tray_items:
+            # access the actual plugin object (there is no store for plugins)
+            tray_obj = self.widgets.get(tray_item['widget'].split('IPY_MODEL_')[1])
+            for bqplot_fig in tray_obj.bqplot_figs_resize:
+                bqplot_fig.layout.height = '99.9%'
+                bqplot_fig.layout.height = '100&'
```

Comment:
Just checking - is this really supposed to be `100&`, not `100%`?

Reply:
Hmm, @kecnry would know.

Reply:
Good catch, yes it should be `100%`

Reply:
Okay, I pushed a commit to fix it. Makes me wonder what it was doing before. I didn't see anything weird in manual testing. Does this setting even do anything?

Reply:
@kecnry put that in when I noticed the radial plot won't resize when I resize the plugin tray.